docs/reference/coap-api.md

```diff
@@ -21,9 +21,14 @@ ThingsBoard server nodes act as a CoAP Server that supports both regular and obs
 
 ##### Client libraries setup
 
-You can find CoAP client libraries for different programming languages on the web. Examples in this article will be based on [CoAP cli](https://www.npmjs.com/package/coap-cli).
+You can find CoAP client libraries for different programming languages on the web. Examples in this article will be based on [CoAP cli](http://manpages.ubuntu.com/manpages/focal/man5/coap-client.5.html).
```

Comment:
Keep previous link and add new one. Don't replace https://www.npmjs.com/package/coap-cli with coap-client!

Reply:
@ashvayka done.